index.html

```diff
@@ -7,5 +7,14 @@
     <h1>Project Night Projects from Code for San Jose</h1>
     <p>Small projects for new members to work on.
     <p><a href="instructions.html">Instructions for new members</a>
+    <br>
+    <p><u>Projects in Progress</u></p>
```

Comment:
Feel free to make the changes.


Date: Wed, 7 Oct 2015 06:54:22 -0700
From: notifications@github.com
To: HackNightProjects@noreply.github.com
CC: sagarwal10@hotmail.com
Subject: Re: [HackNightProjects] Added links for San Jose zoning data using CartoDB (#8)

In index.html:

> @@ -7,5 +7,14 @@
>      <h1>Project Night Projects from Code for San Jose</h1>
>      <p>Small projects for new members to work on.
>      <p><a href="instructions.html">Instructions for new members</a>
> +    <br>
> +    <p><u>Projects in Progress</u></p>

Could you wrap "projects in progress" in an H2 tag instead of  P and U tags?


—
Reply to this email directly or view it on GitHub.